### ISSUE TITLE
Adding support to scrape HTML tables

### ISSUE THREAD

**Issue by spekulatius** [spekulatius/PHPScraper#152]
Could be implemented similar to CSV (simple & header).

**Comment by mpathy** [comment]
+1 
I mean, this is the first thing everybody looks for in a web scraper.
This is the most common source for data that someone wants to scrape.

**Comment by spekulatius** [comment]
Yeah, definitely is an important feature. Are you keen to make a start?
